lib/pact/provider/pact_spec_runner.rb

```diff
@@ -131,6 +132,8 @@ def initialize_specs
             request_customizer: options[:request_customizer]
           }
           honour_pactfile pact_source, ordered_pact_json(pact_source.pact_json), spec_options
+
+          Pact::Utils::Metrics.report_metric("Pacts Verified", "ProviderTest", "Completed")
```

Comment:
Minor, but for consistency with the other metrics:

```suggestion
          Pact::Utils::Metrics.report_metric("Pacts verified", "ProviderTest", "Completed")
```